### PR TITLE
More installer streamlining

### PR DESCRIPTION
**Pull request by zanderz**
Making the json by hand is tedious and error prone. Not sure this kind of stuff needs to be reviewed?
@maxtaco @gabriel

### CODE REVIEW

File: go/winresource/UpdateJson.go [hunk 1]
```diff
@@ -0,0 +1,71 @@
...
+
+	var update = keybase1.Update{
+		Version:     ver,
+		Name:        "v" + ver,
+		Description: "",
+		Type:        keybase1.UpdateType_NORMAL,
+		PublishedAt: &publishedAt,
+		Asset: &keybase1.Asset{
+			Name:      fname,
+			Url:       "https://s3.amazonaws.com/prerelease.keybase.io/windows-updates/" + strings.Replace(fname, "+", "%2B", -1),
```
Comment (maxtaco) [inline go/winresource/UpdateJson.go]: can you use the Go URL-encoding library for this?

**Comment by maxtaco** [comment]
one small comment. Good idea, :+1: !

File: go/winresource/UpdateJson.go (hunk 1 above)
Comment (zanderz) [inline go/winresource/UpdateJson.go]: It doesn't work. Tried:
```
u, err := url.Parse("https://s3.amazonaws.com/prerelease.keybase.io/windows-updates/" + fname)
...
Url:       u.EscapedPath(),
```
and also:
`Url:       u.String(),`

It's still a plus sign in both cases.

Comment (maxtaco) [inline go/winresource/UpdateJson.go]: ok,  then fahgeddddaboutit

**Comment by gabriel** [comment]
is the file supposed to be CamelCase? Probably rename that to update_json.go?

**Comment by gabriel** [comment]
Also there is a script to do update json output in the release tool:
https://github.com/keybase/release

**Comment by gabriel** [comment]
Opened issue to use that here:
https://keybase.atlassian.net/browse/DESKTOP-515

**Pull request merged**